src/index.js

```diff
@@ -2,6 +2,7 @@ import React from 'react';
 import ReactDOM from 'react-dom/client';
 import './index.css';
 import App from './App';
+
 import reportWebVitals from './reportWebVitals';
```

Comment:
- [ ] Kindly remove all CRA boilerplate in your project as they are not related to this project and would help make your code more professional. There are still some boilerplate files in your ```public```  and ```src``` folders. Please go through your codebase and remove them.